Refresh.GameServer/Endpoints/Game/DataTypes/Request/GameLevelRequest.cs

```diff
@@ -43,7 +43,9 @@ public class GameLevelRequest
     [XmlElement("initiallyLocked")] public bool IsLocked { get; set; }
     [XmlElement("isSubLevel")] public bool IsSubLevel { get; set; }
     [XmlElement("shareable")] public int IsCopyable { get; set; }
-
+    
+    [XmlElement("backgroundGUID")] public string? BackgroundGuid { get; set; }
```

Comment:
Do we know what the format of this is? Is it ever actually used? Thinking about ways we could sanitize this/whether we need it at all

Reply:
I believe it is just a `GUID` asset ID (`gXXXXX`), however i dont believe ive actually seen the game upload levels with this `backgroundGUID` field set yet, i just saw it when RE and figured "might as well send it back to make the game happy", i'll do a deeper dive later

Reply:
I believe mainline can sometimes send this too, but I haven't nailed down a pattern on when/where. We can address this after merging.